src/core/components/fields/LabeledTextAreaField.tsx

```diff
@@ -43,7 +47,7 @@ export const LabeledTextAreaField = forwardRef<HTMLTextAreaElement, LabeledTextA
           </div>
         )}
 
-        <style jsx>{`
+        <style jsx="true">{`
           label {
```

Comment:
Is there a reason to make the jsx = true here? I had to take it out before because it causes errors in compiling.

Specifically here --> is there a reason to change the base component when working on the test?

Reply:
Oh or was this a change because you started this before we pushed?

Reply:
We should have a build test on each PR to help identify these issues, because any changes to make tests work but make the build fail defeat their purpose of securing the build. This will be resolved by #333 once that's merged.

@doomlab this is a thing that's been introduced in the test branches and I am surprised it affects the build as it looks like such an innocuous flag. My apologies for approving this and leading to build issues for you!

Reply:
It is a little bit strange because before the PR I ran blitz dev, and the apps worked correctly. In future tests I will  also check with npm run build.

Reply:
@chartgerink @doomlab 
I created the branch https://github.com/STAPLE-verse/STAPLE/tree/fixing_jsx_true to investigate the problem. I found some discussions with similar issues but the fixes that they propose do not work when I tested them. The links are included in the commit message. 

One solution that works (not warnings and compiles) is to remove the **jsx** from the style tag. After running the application, the styling of the components seem to be the same, but it will be good if another person also validates it.